starter-code/scripts/article.js

```diff
@@ -6,15 +6,23 @@ function Article (opts) {
   // TODO: Use the JS object passed in to complete this constructor function:
   // Save ALL the properties of `opts` into `this`
   // lots of this.someProperty = opts.property
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.title = opts.title;
+  this.body = opts.body;
+  this.publishedOn = opts.publishedOn;
+
 }
 
+
 Article.prototype.toHtml = function() {
   var $newArticle = $('article.template').clone();
   /* TODO: This cloned article is no longer a template,
   as it now has real data attached to it! We need to account
   for that before this current article gets rendered to our
   DOM. */
   // one line of code to remove the class that hides the template so that the articles will actually show
+  $newArticle.removeClass('template');
```

Comment:
Excellent! Note that you can also chain this .removeClass() to the end of line 19, and still achieve the same result.